apps/api-server/src/routes/api/project.js

```diff
@@ -302,9 +302,24 @@ router.route('/:projectId') //(\\d+)
     req.pendingMessages = [{ key: `project-${project.id}-update`, value: 'event' }];
     if (req.body.url && req.body.url != project.url) req.pendingMessages.push({ key: `project-urls-update`, value: 'event' });
 
+    // Update allowedDomains if creating a new site
+    let updateBody = req.body;
+    if((project?.config?.widgets?.allowedDomains || []).length === 0 && req?.body?.url){
+      updateBody.config = updateBody.config || {};
+      updateBody.config.widgets = updateBody.config.widgets || {};
+      updateBody.config.widgets.allowedDomains = [req.body.url];
```

Comment:
Deze url moet nog worden aangepast; 
- Mag geen protocol bevatten
- Path moet eraf gestript worden

Dan houden we alleen domein + TLD over (bijv `openstad.org`) die we dan in de allowedDomains gebruiken.

Reply:
✅